source/docs/software/roborio-info/roborio-introduction.rst

```diff
@@ -0,0 +1,10 @@
+roboRIO Introduction
+====================
+
+The roboRIO is designed specifically with FIRST in mind. The roboRIO uses the same basic architecture of a Real-Time processors + FPGA as the previous cRIO but is more powerful, lighter, and smaller.
+
+roboRIO - Advanced Robotics Controller—The roboRIO is a reconfigurable robotics controller that includes built-in ports for inter-integrated circuits (I2C), serial peripheral interfaces (SPI), RS232, USB, Ethernet, pulse width modulation (PWM), and relays to quickly connect the common sensors and actuators used in robotics. The controller features LEDs, buttons, an onboard accelerometer, and a custom electronics port. It has an onboard dual-core ARM real-time Cortex‑A9 processor and customizable Xilinx FPGA.
+
+Detailed information on the roboRIO can be found in the `roboRIO User Manual <https://www.ni.com/pdf/manuals/374474a.pdf>`__ and in the `roboRIO technical specifications <https://www.ni.com/pdf/manuals/374661a.pdf>`__.
+
+Before deploying programs to your roboRIO, you must first image the roboRIO with the FRC roboRIO Imaging Tool.
```

Comment:
I know this is the next article but maybe we should say that here more explicitly so the user isn't wondering why we didn't link it in this sentence?

Reply:
Linked and mentioned next article